src/microbe_stage/organelle_unlocks/ConditionSet.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace UnlockConstraints
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    public class ConditionSet : IUnlockCondition
```

Comment:
I think I can do this when fixing other JSON stuff but making this a generic collection would allow the JSON in organelles to avoid one level of extra wrapping for simpler more understandable way to specify the conditions.

Reply:
This is not needed anymore as I've implemented a custom Json Converter